include/boost/url/url_view.hpp

```diff
@@ -1700,6 +1701,30 @@ class BOOST_SYMBOL_VISIBLE url_view
             opt, a, decoded_[id_frag]);
     }
 
+    //--------------------------------------------
+    //
+    // Comparison
+    //
+    //--------------------------------------------
+
+    /** Compare two URLs.
+
+        This function compares two algorithms
```

Comment:
you are saying algorithm twice, and the usage of the first occurence is wrong. We are comparing two URLs not two algorithms...